src/shared/components/mutationTable/column/AnnotationColumnFormatter.tsx

```diff
@@ -400,6 +404,23 @@ export default class AnnotationColumnFormatter {
                         userEmailAddress={columnProps.userEmailAddress}
                     />
                 </If>
+                <If
+                    condition={
+                        (columnProps.enableClinicalTrials &&
+                            columnProps.enableOncoKb &&
+                            annotation.oncoKbGeneExist &&
+                            annotation.isOncoKbCancerGene) ||
+                        false
+                    }
+                >
+                    <ClinicalTrialsCard
+                        hugoGeneSymbol={annotation.hugoGeneSymbol}
+                        status={annotation.oncoKbStatus}
+                        hideClosedTrials={true}
```

Comment:
The clinical trial card displays all trials by default. If users want to see active trials only, please set `hideClosedTrials` to true.